### DISCUSSION TITLE
Customizing Window Title

### DISCUSSION THREAD

**Discussion by Topener** [docsifyjs/docsify#1611] [Q&A] [answered]
Right now when I customize the window title it only shows up on the homepage, but when I go to another page the window title takes the title of the page itself. I'm assuming this is the default behaviour.

How can I format the window title? I'd prefer to have a prefix and/or suffix to the title so I can add the name of the documentation to the title as well.

**Comment by sy-records** [comment, marked as answer]
you can use :

```markup
window.$docsify = {
  plugins: [
    function(hook, vm) {
      hook.doneEach(function() {
        // Invoked each time after the data is fully loaded, no arguments,
       document.title += ' - title';
      });
    }
  ]
};
```

**Reply by Topener** [reply]
Perfect, thanks so much!
